tests/files/rbdigital/response_patron_checkouts_200_list.json

```diff
@@ -71,39 +71,39 @@
     "images": [
       {
         "name": "xx-small",
-        "url": "http://images.oneclickdigital.com/78226/78226_image_32x50.jpg"
+        "url": "http://images.rbdigital.com/78226/78226_image_32x50.jpg"
       },
       {
         "name": "x-small",
-        "url": "http://images.oneclickdigital.com/78226/78226_image_37x56.jpg"
+        "url": "http://images.rbdigital.com/78226/78226_image_37x56.jpg"
       },
       {
         "name": "small",
-        "url": "http://images.oneclickdigital.com/78226/78226_image_71x108.jpg"
+        "url": "http://images.rbdigital.com/78226/78226_image_71x108.jpg"
       },
       {
         "name": "medium",
-        "url": "http://images.oneclickdigital.com/78226/78226_image_95x140.jpg"
+        "url": "http://images.rbdigital.com/78226/78226_image_95x140.jpg"
       },
       {
         "name": "large",
-        "url": "http://images.oneclickdigital.com/78226/78226_image_128x192.jpg"
+        "url": "http://images.rbdigital.com/78226/78226_image_128x192.jpg"
       },
       {
         "name": "x-large",
-        "url": "http://images.oneclickdigital.com/78226/78226_image_148x230.jpg"
+        "url": "http://images.rbdigital.com/78226/78226_image_148x230.jpg"
       },
       {
         "name": "xx-large",
-        "url": "http://images.oneclickdigital.com/78226/78226_image_512x512_iTunes.png"
+        "url": "http://images.rbdigital.com/78226/78226_image_512x512_iTunes.png"
       }
     ],
     "titleId": 0,
     "isbn": "9781456103859",
     "title": "The Greatest: Muhammad Ali",
     "publisher": "Recorded Books, Inc.",
     "expiration": "2016-11-19",
-    "output": "Due to Microsoft's decision to no longer support DRM encryption, protected OneClickdigital titles can no longer be downloaded as .zip files. You can still listen to protected titles in the OneClickdigital Media Manager, stream the titles from your library’s OneClickdigital website, or listen to the title with one of the OneClickdigital mobile apps on Apple, Android, or Kindle Fire devices.",
+    "output": "Due to Microsoft's decision to no longer support DRM encryption, protected rbdigital titles can no longer be downloaded as .zip files. You can still listen to protected titles in the rbdigital Media Manager, stream the titles from your library’s rbdigital website, or listen to the title with one of the rbdigital mobile apps on Apple, Android, or Kindle Fire devices.",
```

Comment:
Change this line back -- we don't know what the output is, but it's more likely what it was before the change.